resources/labelers/column_name_labeler/data_labeler_parameters.json

```diff
@@ -0,0 +1 @@
+{"model": {"class": "ColumnNameModel"}, "preprocessor": {"class": "DirectPassPreprocessor"}, "postprocessor": {"class": "ColumnNamePostprocessor"}}
```

Comment:
these files look good

Reply:
dope

Reply:
typo in here actually I found when `load_from_library` test in the `test_integration` was added